account_invoice_comment_template/migrations/12.0.2.0.0/pre-migrate.py

```diff
@@ -0,0 +1,52 @@
+import logging
+_logger = logging.getLogger(__name__)
+
+
+def migrate(cr, version):
```

Comment:
This is better put in `base_comment_template` checking if the column exists, as other possible default values can ruin this.